packages/features/sales-escrows/modals/StartEscrowModal.tsx

```diff
@@ -297,7 +297,7 @@ export function StartEscrowModal({
 
   return (
     <div>
-      <Modal isOpened={open} closeModal={() => onCustomClose} size="md">
+      <Modal isOpened={open} closeModal={() => onClose(false)} size="md">
```

Comment:
Looks like we still need to call onCustomClose to reset the component state. The issue is that its called via an arrow function with no parens.

Reply:
```suggestion
      <Modal isOpened={open} closeModal={() => onCustomClose(false)} size="md">
```